lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -339,38 +363,11 @@ spec = do
                     (.> (Quantity 0))
                 ]
 
--- TODO: Check if we can enable this
---        -- Quit a pool
---        quitStakePool @n ctx (w, fixturePassphrase) >>= flip verify
---            [ expectResponseCode HTTP.status202
---            , expectField (#status . #getApiT) (`shouldBe` Pending)
---            , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
---            ]
---        eventually "Certificates are inserted after quiting a pool" $ do
---            let ep = Link.listTransactions @'Shelley w
---            request @[ApiTransaction n] ctx ep Default Empty >>= flip verify
---                [ expectListField 0
---                    (#direction . #getApiT) (`shouldBe` Outgoing)
---                , expectListField 0
---                    (#status . #getApiT) (`shouldBe` InLedger)
---                , expectListField 1
---                    (#direction . #getApiT) (`shouldBe` Outgoing)
---                , expectListField 1
---                    (#status . #getApiT) (`shouldBe` InLedger)
---                ]
---
---        -- Check that rewards have stopped flowing.
---        waitForNextEpoch ctx
---        waitForNextEpoch ctx
---        reward <- getFromResponse (#balance . #getApiT . #reward) <$>
---            request @ApiWallet ctx (Link.getWallet @'Shelley w) Default Empty
---
---        waitForNextEpoch ctx
---        request @ApiWallet ctx (Link.getWallet @'Shelley w) Default Empty >>= flip verify
---            [ expectField
---                    (#balance . #getApiT . #reward)
---                    (`shouldBe` reward)
---            ]
+        -- Can't quite if unspoiled rewards.
```

Comment:
```suggestion
        -- Can't quit if unspoiled rewards.
```
But also, why not "Can't quit with remaining rewards.", or even "Can't quit with unredeemed rewards."?

The term "unspoiled" seems a bit weird to me at least.